python/ray/data/tests/preprocessors/test_tokenizer.py

```diff
@@ -24,7 +24,56 @@ def test_tokenizer():
     ]
     expected_df = pd.DataFrame.from_dict({"A": processed_col_a, "B": processed_col_b})
 
-    assert out_df.equals(expected_df)
+    pd.testing.assert_frame_equal(out_df, expected_df, check_like=True)
+
+    # Test append mode
+    with pytest.raises(ValueError):
+        Tokenizer(columns=["A", "B"], output_columns=["A_tokenized"])
```

Comment:
check error message pls?

Reply:
@richardliaw there's no other error message being thrown, do you still think matching the error message is necessary?

Reply:
added the match anyways